Cargo.toml

```
@@ -736,7 +736,8 @@ sinks-datadog_metrics = ["protobuf-build", "dep:prost-reflect"]
 sinks-datadog_traces = ["protobuf-build", "dep:rmpv", "dep:rmp-serde", "dep:serde_bytes"]
 sinks-elasticsearch = ["transforms-metric_to_log"]
 sinks-file = ["dep:async-compression"]
-sinks-gcp = ["dep:base64", "gcp"]
+sinks-gcp = ["sinks-gcp-chronicle"]
+sinks-gcp-chronicle =  ["dep:base64", "gcp"]
```

```suggestion
sinks-gcp = ["sinks-gcp-chronicle", dep:base64", "gcp"]
sinks-gcp-chronicle =  ["gcp"]
```

Nit: I don't think `sinks-gcp-chronicle` actually depends on `base64`, only `gcp_pubsub` does.

---

Correct, committed this suggestion (and the typo fix).